aopy/visualization.py

```diff
@@ -277,6 +278,42 @@ def plot_spatial_map(data_map, x, y, alpha_map=None, ax=None, cmap='bwr'):
 
     return image
 
+def plot_ECoG244_data_map(data, bad_elec=[], interp=True, cmap='bwr', ax=None, **interp_kwargs):
+    '''
+    Plot a spatial map of data from an ECoG244 electrode array from the Viventi lab.
+
+    Args:
+        data ((256,) array): values from the ECoG array to plot in 2D
+        bad_elec (list, optional): channels to remove from the plot. Defaults to [].
+        interp (bool, optional): flag to include 2D interpolation of the result. Defaults to True.
+        cmap (str, optional): matplotlib colormap to use in image. Defaults to 'bwr'.
+        ax (pyplot.Axes, optional): axis on which to plot. Defaults to None.
+
+    Returns:
+        pyplot.Image: image returned by pyplot.imshow. Use to add colorbar, etc.
+    '''
+    
+    if ax is None:
+        ax = plt.gca()
+    
+    # Load the signal path files
+    elec_pos, acq_ch, elecs = load_chmap(drive_type='ECoG244')
+
+    # Remove bad electrodes
+    bad_ch = acq_ch[np.isin(elecs, bad_elec)]-1
```

Comment:
I've been declaring acq_ch_idx and bad_ch_idx because I kept forgetting to subtract the 1 when indexing. Might be good to do here!